src/util/system.h

```diff
@@ -62,7 +62,6 @@ int64_t GetStartupTime();
 static const int64_t DEFAULT_POW_CACHE_SIZE = 1000000;
 static const int DEFAULT_MAX_LOAD_SIZE = 720;
 static bool DEFAULT_VALIDATE_POW_CACHE = false;
-static std::string DEFAULT_IPFS_SERVICE_URL = "https://ipfsm.raptoreum.com/";
```

Comment:
It is used here but not sure how ready this HTTP client is already. [src/qt/httpclient.cpp@21](https://github.com/Raptor3um/raptoreum/pull/390/files#diff-afeef772ed464993a8708065a59bf26de3ed31f256116e98019240cafb875867R21)

Reply:
httpclient is not include in the build atm. i just keep it here for reference later if i want to change to different implementation,